src/components/OptionsTiles/OptionTiles.tsx

```diff
@@ -0,0 +1,330 @@
+import React from 'react';
+import { BrandColor, FontColor } from '../../types';
+import { Box, BoxProps } from '../Box/Box';
+import { RadioInput } from '../RadioGroup/RadioInput/RadioInput';
+import { Checkbox } from '../CheckboxInput/components/Checkbox';
+import { InputValidationMessage } from '../InputValidationMessage/InputValidationMessage';
+import { Icon } from '../Icon/Icon';
+import styles from './OptionTiles.module.scss';
+
+interface Option {
+  id: string;
+  value: string;
+  label: string;
+  disabled?: boolean;
+  render?: (option: {
+    id: string;
+    value: string;
+    label: string;
+    disabled?: boolean;
+  }) => React.ReactNode;
+}
+
+export interface OptionTilesProps extends BoxProps {
+  /**
+   * Option group name (to be passed to either radio or checkbox inputs)
+   */
+  name: string;
+  /**
+   * Change event.
+   */
+  onChange: (event: React.ChangeEvent<HTMLInputElement> | React.MouseEvent<HTMLDivElement>) => void;
+  /**
+   * Options to display
+   */
+  options: Option[];
+  /**
+   * Value of selected option(s).
+   */
+  value: null | string | number | (string | number)[];
+  /**
+   * Direction (flex direction) for option tiles.
+   */
+  direction?: BoxProps['direction'];
```

Comment:
Do we need this if it's already included in `BoxProps`? Does a value of `row-reverse` and `column-reverse` work?

Reply:
@nathanyoung I added it mostly because I found it to be a core modifier for anybody using these. In other words, this would make it into this component even if it was not extending the Box, so I think it's worth a call out and a place at the top tier of the docs.